### DISCUSSION TITLE
Добавление новых сайтов

### DISCUSSION THREAD

**Discussion by Flaanumans** [ilyhalight/voice-over-translation#520] [Q&A] [answered]
Хотелось бы знать можно ли добавлять сайты не указанные в списке, и если можно то каким условиям они должны соответствовать?

**Comment by ilyhalight** [comment, marked as answer]
При добавление сайта ориентируемся на [этот список](https://github.com/yt-dlp/yt-dlp/tree/master/yt_dlp/extractor), если сайт есть в списке, то с большой вероятностью он заработает без лишних танцев с бубном. Во всех остальных случаях все индивидуально, но необходимо, чтобы сайт хранил видео в mp4/webm

**Reply by lionevil1** [reply]
В списке есть указан сайт teachable.com но ни как не могу завести на нем перевод

**Reply by ilyhalight** [reply]
> В списке есть указан сайт teachable.com но ни как не могу завести на нем перевод

Если он есть в списке, то это не значит, что он уже добавлен в расширение

**Comment by FiLinX** [comment]
там вот это, появился типа прямой аналог ютуба, сервис "Платформа" - https://plvideo.ru/
возможно будет так же легко натравить VOT и на него)
кое что для [как получить линки потоков](https://forum.ru-board.com/topic.cgi?forum=5&topic=51339&start=360#19)

**Reply by ilyhalight** [reply]
так легко не получится

**Reply by ilyhalight** [reply]
> так легко не получится

Можешь в issues закинуть, чтобы не забыл
